source/releasenotes/2024-05-10-sendgrid-transactional-emails.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Transactional Emails: Rest API emails now sent via SendGrid effective May 12, 2024"
+published_date: "2024-05-10"
+categories: [new-feature] [WordPress] [Drupal]
+---
+
+Effective May 12th - We will begin rolling out SendGrid to send transactional emails.
+* Pantheon will begin using SendGrid to send customer transactional emails via rest API during the week of May 12th, 2024 (we expect the full cutover to be complete by the end of May).
```

Comment:
```suggestion
* Pantheon will begin using SendGrid to send customer transactional emails via REST API during the week of May 12th, 2024 (we expect the full cutover to be complete by the end of May).
```

REST should be capitalized. But is that a relevant detail for customers? Or do they just need to know that emails will now come from SendGrid for messages originating from WordPress or Drupal?

Reply:
How about this?
```suggestion
* Pantheon will begin using SendGrid to send during the week of May 12th, 2024 (we expect the full cutover to be complete by the end of May).
```